src/App.css

```diff
@@ -36,15 +36,14 @@
     display: flex;
     flex-direction: row;
     justify-content: space-evenly;
-    margin-block-start: 20px;
+    margin-block-start: 1rem;
 }
 
 .vs {
     font-size: 2rem;
     font-weight: bold;
-    margin-block-end: 60px;
-    margin-left: 20px;
-    margin-right: 20px;
+    padding: 0.7em;
+    margin-inline-start: 0.1em;
```

Comment:
Is the asymmetrical margin intentional?

Reply:
It was to make the containers spreaded a bit wider, but I can just make the padding 1em and have no margin